Assets/Scripts/Game/UserInterface/UpDownSpinner.cs

```diff
@@ -121,9 +140,13 @@ public UpDownSpinner(Rect spinnerScreenRect, Rect upButtonRect, Rect downButtonR
             upButton.Position = new Vector2(upButtonRect.xMin, upButtonRect.yMin);
             upButton.Size = new Vector2(upButtonRect.width, upButtonRect.height);
             upButton.OnMouseClick += UpButton_OnMouseClick;
+            upButton.OnMouseDown += UpButton_OnMouseDown;
```

Comment:
Why does this class have 2 almost identical constructors Interkarma? If there's no special reason then I could combine the code if you like.

Reply:
Hmm, not sure how that came about. Happy for these to be combined. :)